frontend/src/components/HistoVecFooter.vue

```diff
@@ -64,48 +53,78 @@ export default defineComponent({
-  >
-    <template #description>
-      <div>
-        <p>
-          HistoVec : un service proposé par la délégation à la sécurité routière.
-        </p>
-        <p>
-          Jouez la transparence : partagez l'historique de votre véhicule.
-        </p>
+  <footer class="fr-footer" role="contentinfo" id="footer">
+    <div class="fr-container">
+      <div class="fr-footer__body">
+        <div class="fr-footer__brand fr-enlarge-link">
+          <p class="fr-logo">
+            Ministère
+            <br>de l’intérieur
+          </p>
+          <router-link class="fr-footer__brand-link" to="/" title="Ministère de l’intérieur - HistoVec - Retour à l’accueil du site">
+            <img class="fr-footer__logo" style="width:7.5rem;" :src='images.logoHistoVecSvg' alt="" />
+          </router-link>
+        </div>
+        <div class="fr-footer__content">
+          <p class="fr-footer__content-desc">HistoVec : un service proposé par la délégation à la sécurité routière.
+          </p>
+          <p class="fr-footer__content-desc">Jouez la transparence : partagez l'historique de votre véhicule.
+          </p>
+          <ul class="fr-footer__content-list">
+            <li v-for="(contentLink, index) in contentLinks"
+                :key="index"
+                class="fr-footer__content-item"
+            >
+              <a class="fr-footer__content-link" target="_blank" :href="contentLink.href">{{ contentLink.label }}</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div class="fr-footer__partners">
+        <h4 class="fr-footer__partners-title">Nos partenaires</h4>
+        <div class="fr-footer__partners-logos">
+          <div class="fr-footer__partners-main">
+            <a id="securiteRoutiereIconeImage" class="fr-footer__partners-link" rel="noopener noreferrer" target="_blank" href="https://securite-routiere.gouv.fr" title="Securité routière - Lien vers le site securite-routiere.gouv.fr">
+              <img class="fr-footer__logo" style="height: 5.625rem" :src='images.logoSecuriteRoutiereMb90Svg' alt="" />
+            </a>
+          </div>
+        </div>
+      </div>
+      <div class="fr-footer__bottom">
+        <ul class="fr-footer__bottom-list">
+          <li v-for="(mandatoryLink, index) in mandatoryLinks"
+              :key="index"
+              class="fr-footer__bottom-item"
+          >
+            <router-link class="fr-footer__bottom-link" :to="mandatoryLink.to">{{ mandatoryLink.label }}</router-link>
+          </li>
+        </ul>
+        <div class="fr-footer__bottom-copy">
+          <p>Sauf mention contraire, tous les contenus de ce site sont sous <a href="https://github.com/etalab/licence-ouverte/blob/master/LO.md" target="_blank">licence etalab-2.0</a>
+          </p>
+        </div>
       </div>
-    </template>
-  </DsfrFooter>
+    </div>
+  </footer>
 </template>
+<style scoped>
 
+#securiteRoutiereIconeImage[target=_blank]:after {
+  content: '';
```

Comment:
Avec un content '', il y a quand meme un after mais il n'est pas visible car pas de taille définit. C'est mieux de completement le retirer.

```suggestion
  content: none;
```